src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -175,6 +175,15 @@ public static BigIntUnsignedHandlingMode parse(String value, String defaultValue
         }
     }
 
+    public EventProcessingFailureHandlingMode getEventProcessingFailureHandlingMode() {
```

Comment:
This should be unnecessary as `io.debezium.config.CommonConnectorConfig.EVENT_PROCESSING_FAILURE_HANDLING_MODE` declares `FAIL` as the default value.